plugin/yt_dlp_plugins/extractor/getpot_bgutil_script.py

```diff
@@ -6,41 +6,69 @@
 
 from yt_dlp.networking.exceptions import RequestError, UnsupportedRequest
 from yt_dlp.utils import Popen, classproperty
-from yt_dlp_plugins.extractor.getpot import GetPOTProvider, register_provider, register_preference
+from yt_dlp_plugins.extractor.getpot import (
+    GetPOTProvider,
+    register_provider,
+    register_preference,
+)
 from yt_dlp_plugins.extractor.getpot_bgutil import __version__
 
 
 @register_provider
 class BgUtilScriptPotProviderRH(GetPOTProvider):
     _PROVIDER_NAME = 'BgUtilScriptPot'
-    _SUPPORTED_CLIENTS = ('web', 'web_safari', 'web_embedded', 'web_music', 'web_creator')
+    _SUPPORTED_CLIENTS = (
+        'web',
+        'web_safari',
+        'web_embedded',
+        'web_music',
+        'web_creator',
+    )
     VERSION = __version__
 
     @classproperty(cache=True)
     def _default_script_path(self):
         home = os.path.expanduser('~')
         return os.path.join(
-            home, 'bgutil-ytdlp-pot-provider', 'server', 'build', 'generate_once.js')
+            home, 'bgutil-ytdlp-pot-provider', 'server', 'build', 'generate_once.js'
+        )
```

Comment:
this is intentional:
> Always move trailing parentheses used for grouping/functions after the last argument. On the other hand, multi-line literal list/tuple/dict/set should closed be in a new line. Generators and list/dict comprehensions may use either style

Reply:
All this source code changes in this PR are from the respective linters. Will see if I can copy yt-dlp's ruff config to match the behavior